### DISCUSSION TITLE
Does delayed submission have an impact on farming rewards

### DISCUSSION THREAD

**Discussion by Michael121318** [Chia-Network/chia-blockchain#16707] [Support]
![image](https://github.com/Chia-Network/chia-blockchain/assets/147701612/b90b8dfd-25a9-45f6-8cd5-9c48849a5f27)

**Comment by BrandtH22** [comment]
Hey @Michael121318 , as a heads up we can generally provide more timely, thorough support on our discord server (https://discord.gg/chia).

As a quick answer having a few late partials or missing signage points is generally not an issue as most farmers have ~ 9200 signage points in a day and a comparable number of partials so 100 missing is ~ 1% and generally related to network or hardware issues (such as an ISP issuing new IPs can result in a few missing signage points)

**Reply by hajes** [reply]
that is interesting question very often discussed. my theory is that FIFO (First In First Out) as in the good old crypto mining days.

If three guys fight for same block, fastest win no? Block cannot be split among more people.

**Reply by BrandtH22** [reply]
Nope, chia uses something similar to bitcoins heaviest chain rule meaning more than 1 farmer can farm the same block but these "forks" are quickly resolved as one has a larger weight and the other(s) collapse into the heaviest chain.

More information can be found here:
https://docs.chia.net/overflow-blocks#weight

**Reply by hajes** [reply]
As usually, the reference manuals are not for normal people - very few understand it ROFL

There is written something about attack, but is unclear the dynamics of block wars. What I understand is if there is a block win collision, it is resolved by space-size, perhaps time.

I have never received a partial reward for found blocks. My guess is fastest/largest wins.

Care to collaborate, and explain it for normal people, please?

**Reply by BrandtH22** [reply]
The consensus docs do generally require more technical understanding to grasp the concepts in full.

Chia uses something similar to bitcoins heaviest chain rule meaning more than 1 farmer can farm the same block but these "forks" are quickly resolved as one has a larger weight and the other(s) collapse into the heaviest chain.

So there can be multiple winners of the same block each receiving the full reward but these "fork chains" that are created due to multiple winners are quickly collapsed into a single chain again based on the weight of the chains, the heaviest (meaning the one with the most transactions and other block records) will be the main chain.

Feel free to jump into our discord to more thoroughly discuss (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
